rules/windows/process_creation/proc_creation_win_screenconnect_remote_command_execution.yaml

```diff
@@ -0,0 +1,26 @@
+title: Remote Access Tool - ScreenConnect Remote Command Execution
+id: b1f73849-6329-4069-bc8f-78a604bb8b23
+status: experimental
+description: Detect the execution of system command via ScreenConnect RMM
+author: Ali Alwashali
+date: 2023/10/10
+tags:
+    - attack.execution
+    - attack.t1059.003
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_parent:
+        ParentImage|endswith: '\ScreenConnect.ClientService.exe'
+    selection_img:
+        - Image|endswith: '\cmd.exe'
+        - OriginalFileName: 'Cmd.Exe'
+    selection_cli:
+        #Example:
+        #   CommandLine: "cmd.exe" /c "C:\Windows\TEMP\ScreenConnect\23.6.8.8644\3c41d689-bbf5-4216-b2f4-ba8fd6192c25run.cmd"
+        CommandLine|contains: '\TEMP\ScreenConnect\'
```

Comment:
Since this is executing from the service, it'll always drop at the admin temp. Can the location be changed?

Reply:
As far as I know, it can't be changed. I didn't find options around changing it.